lodash.src.js

```diff
@@ -7325,25 +7325,17 @@
      * _.map(_.sortByAll(users, ['user', 'age']), _.values);
      * // => [['barney', 26], ['barney', 36], ['fred', 30], ['fred', 40]]
      */
-    function sortByAll() {
```

Comment:
I mean I was hesitant to include it as well because of it's inclusion in
ES6 but I'm glad regardless

On Sun, Apr 12, 2015 at 1:18 AM, John-David Dalton <notifications@github.com
> wrote:

> In lodash.src.js
> <https://github.com/lodash/lodash/pull/1125#discussion_r28200808>:
>
> > @@ -7325,25 +7325,17 @@
> >       * _.map(_.sortByAll(users, ['user', 'age']), _.values);
> >       * // => [['barney', 26], ['barney', 36], ['fred', 30], ['fred', 40]]
> >       */
> > -    function sortByAll() {
>
> Adam is talking out his ass over it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lodash/lodash/pull/1125/files#r28200808>.
>